2022-08-22 10:21:21,270 INFO    MainThread:4464 [wandb_setup.py:_flush():76] Configure stats pid to 4464
2022-08-22 10:21:21,270 INFO    MainThread:4464 [wandb_setup.py:_flush():76] Loading settings from C:\Users\wdrdg\.config\wandb\settings
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_setup.py:_flush():76] Loading settings from E:\Project\Generalized_MADDPG\wandb\settings
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'maml_test.py', 'program': 'E:/Project/Generalized_MADDPG/maml_test.py'}
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_init.py:_log_setup():461] Logging user logs to E:\Project\Generalized_MADDPG\wandb\run-20220822_102121-2kv3sgbl\logs\debug.log
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_init.py:_log_setup():462] Logging internal logs to E:\Project\Generalized_MADDPG\wandb\run-20220822_102121-2kv3sgbl\logs\debug-internal.log
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_init.py:init():495] calling init triggers
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_init.py:init():499] wandb.init called with sweep_config: {}
config: {}
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_init.py:init():548] starting backend
2022-08-22 10:21:21,271 INFO    MainThread:4464 [wandb_init.py:init():552] setting up manager
2022-08-22 10:21:21,278 INFO    MainThread:4464 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-08-22 10:21:21,281 INFO    MainThread:4464 [wandb_init.py:init():558] backend started and connected
2022-08-22 10:21:21,283 INFO    MainThread:4464 [wandb_init.py:init():636] updated telemetry
2022-08-22 10:21:21,310 INFO    MainThread:4464 [wandb_init.py:init():668] communicating run to backend with 30 second timeout
2022-08-22 10:21:24,374 WARNING MsgRouterThr:4464 [router.py:message_loop():76] message_loop has been closed
2022-08-22 10:21:51,314 INFO    MainThread:4464 [interface_shared.py:_communicate_run():331] couldn't get run from backend
2022-08-22 10:21:51,314 ERROR   MainThread:4464 [wandb_init.py:init():676] backend process timed out
2022-08-22 10:21:51,314 ERROR   MainThread:4464 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
For more info see: https://docs.wandb.ai/library/init#init-start-error
